backgroundScript.js

```diff
@@ -22,6 +28,6 @@ chrome.runtime.onInstalled.addListener(() => {
 			body: JSON.stringify(body)
 		})
 			.then((response) => response.json())
-			.then((data) => dataFromAPI = data);
+			.then((data) => dataFromAPI = data); // Store the response data in the dataFromAPI variable.
```

Comment:
This comment is absolutely useless - given that the variable `dataFromAPI` is not even used or declared anywhere.

Instead, just add a `console.debug` here.

(we can do this later)